POLICIES.md

```diff
@@ -42,6 +42,10 @@ References: [[1](https://github.com/exercism/xjava/issues/177#issuecomment-26129
 
 References: [[1](https://github.com/exercism/xjava/issues/178)]
 
+### No `final` classes
```

Comment:
I'd maybe update this to reflect the scope a little better, since we do allow `final` in example solutions. Perhaps something like

```
### Do not make delivered classes final
```

```
### Do not make served classes final
```

```
### Do not make user-facing classes final
```

?